Mage/src/main/java/mage/MageObject.java

```diff
@@ -173,9 +180,36 @@ default void addSubType(Game game, SubType... subTypes) {
         }
     }
 
+    default void copySubType(MageObject mageObject, Game game) {
+        copySubType(mageObject, game, null);
+    }
+
+    default void copySubType(MageObject mageObject, Game game, SubTypeSet subTypeSet) {
+        if (subTypeSet == SubTypeSet.CreatureType || subTypeSet == null) {
+            this.setIsAllCreatureTypes(mageObject.isAllCreatureTypes(game), game);
+        }
+        for (SubType subType : getSubtype(game)) {
+            if (subType.getSubTypeSet() == subTypeSet || subTypeSet == null) {
```

Comment:
I thinks here is an errror.

`copySubType(mageObject, game, null);` call will not copy any subtypes due `|| subTypeSet == null` (e.g. it can copy only setIsAllCreatureTypes).

Reply:
Are you sure? It looks fine to me

Reply:
Explained here. Also found miss `mageObject.`
![shot_210126_035841](https://user-images.githubusercontent.com/8344157/105780836-caf34c00-5f8a-11eb-8654-446eab842326.png)

Reply:
keep going, it's not done yet

Reply:
Argh, never mind, it was fixed already.

Reply:
yeah it had me pulling my hair out for a while